### PR TITLE
Polish/restaurants

### PR DESCRIPTION
**Pull request by soupette**

### CODE REVIEW

File: client/internals/generators/component/index.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ module.exports = {
       choices: () => [
         'Stateless Function',
         'React.PureComponent',
-        'React.Component',
-      ],
+        'React.Component'
+      ]
```
Comment (virginieky) [inline client/internals/generators/component/index.js]: All commas disappeared

File: client/internals/generators/container/index.js [hunk 2]
```diff
@@ -19,8 +19,8 @@ module.exports = {
       choices: () => [
         'Stateless Function',
         'React.PureComponent',
-        'React.Component',
-      ],
+        'React.Component'
```
Comment (virginieky) [inline client/internals/generators/container/index.js]: Same for commas

File: client/src/components/Card/index.js [hunk 3]
```diff
@@ -18,32 +18,31 @@ function Card(props) {
...
+        />
       </div>
       <div className="card-infos">
         <div className="left-infos">
           <H4>{name}</H4>
           <p className="description">
             <Price value={price} />
             &nbsp;•&nbsp;
-            <span>{category.name}</span>
+            <span id="category-name">{category && category.name}</span>
```
Comment (virginieky) [inline client/src/components/Card/index.js]: Does this id exist only for tests ?

File: client/src/components/Grid/index.js [hunk 4]
```diff
@@ -4,19 +4,42 @@
...
-  return <StyledGrid>{props.children}</StyledGrid>;
-}
-
-Grid.defaultProps = {
-  children: null
-};
-Grid.propTypes = {
-  children: PropTypes.node
-};
+const Grid = styled.ul`
```
Comment (virginieky) [inline client/src/components/Grid/index.js]: 👍

File: client/src/components/Img/StyledImg.js [hunk 5]
```diff
@@ -19,7 +19,8 @@ const StyledImg = styled.img`
   ${props =>
     props.type === 'article' &&
     css`
-      // height: ${sizes.article.img.height};   
+      max-height: ${sizes.article.img.height};
```
Comment (virginieky) [inline client/src/components/Img/StyledImg.js]: Will soon disappear :)

File: client/src/containers/RestaurantPage/RenderView.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Slider from '../../components/Slider';
+
+const RenderView = ({ restaurant }) => {
+  const { cover } = restaurant;
```
Comment (virginieky) [inline client/src/containers/RestaurantPage/RenderView.js]: const { name, cover } = restaurant;

File: client/src/components/Card/index.js (hunk 3 above)
Comment (soupette) [reply client/src/components/Card/index.js]: yes, but i Have updated the tests so it's no longer needed

**Pull request merged**